runtime/src/main/kotlin/kotlin/native/internal/RuntimeUtils.kt

```diff
@@ -44,6 +45,10 @@ fun ThrowInvalidReceiverTypeException(klass: KClass<*>): Nothing {
     throw RuntimeException("Unexpected receiver type: " + (klass.qualifiedName ?: "noname"))
 }
 
+@ExportForCppRuntime
+fun CreateForeignException(payload: NativePtr): Throwable
+        = ForeignException(interpretObjCPointerOrNull<Any?>(payload))
```

Comment:
Typically we place all interop-specific code to `Interop/Runtime`.
Also `ForeignException` should probably be moved there too.